src/xenia/kernel/xsession.h

```diff
@@ -50,6 +53,14 @@ enum class XSESSION_STATE : uint32_t {
   DELETED
 };
 
+enum STATE_FLAGS : uint32_t {
+  STATE_FLAGS_NONE,
+  STATE_FLAGS_CREATED,
+  STATE_FLAGS_HOST,
+  STATE_FLAGS_MIGRATED,
+  STATE_FLAGS_DELETED
+};
+
```

Comment:
I def thing something is wrong here.

If you want for example have combination of CREATED and HOST then now you will receive MIGRATED which is not expected. You should explicitly provide values for these flags 0, 1, 2, 4, 8 etc